release-notes-final-updates-15-0 (#17163)
opf/openproject: docs/release-notes/15-0-0/README.md

> ### A new look-and feel for the Activity tab, with better structure, real-time loading messages, emoji reactions, and more

Reviewer: I think I would write either "look and feel" or "look-and-feel"

> ### A new look-and feel for the Activity tab, with better structure, real-time loading messages, emoji reactions, and more

Reviewer: ```suggestion
### A new look-and-feel for the Activity tab, with better structure, real-time loading messages, emoji reactions, and more
```

> A major change in version 15.0 is the overall look-and-feel of the [Activity tab of work packages](../../user-guide/activity/). Users will notice that the Activity tab has a new design with additional options, and that emoji reactions are now enabled. Additionally, new comments will appear directly without having to reload. This also applies to the notification center, where new notifications will appear in real-time.

Reviewer: ```suggestion
A major change in version 15.0 is the overall look-and-feel of the [Activity tab of work packages](../../user-guide/activity/). Users will notice that the Activity tab has a new design with additional options, and that emoji reactions are now enabled. Additionally, new comments will appear directly without having to reload. This also applies to the notification center, where new notifications will appear in real time.
```

Reviewer: @corinnaguenther I would use "real-time comments",  but "comments will appear in real time".

> This is a big bundle of new features that will greatly improve communication and interaction within OpenProject, making it more simple, more effective and more fun.

Reviewer: ```suggestion
This is a big bundle of new features that will greatly improve communication and interaction within OpenProject, making it simpler, more effective and more fun.
```

### Admin interfaces for SAML and OpenID Connect (Enterprise only)

> OpenProject has for a long time supported SAML and OpenID Connect configured through settings or environment variables. With OpenProject 15.0, Enterprise customers will benefit from our new user interface for SAML and OIDC. This means they can now set up integrations between OpenProject and SAML or OpenID connect stacks independently and offer users options for Single Sign-On (SSO). Before 15.0, SaaS customers had to contact the OpenProject support if they wanted custom integrations with their SAML or OpenID connect providers. These new interfaces greatly improve the user experience of adding these providers, which had been a hurdle at the beginning of your work with OpenProject.

Reviewer: ```suggestion
OpenProject has for a long time supported SAML and OpenID Connect configured through settings or environment variables. With OpenProject 15.0, Enterprise customers will benefit from our new user interface for SAML and OIDC. This means they can now set up integrations between OpenProject and SAML or OpenID connect stacks independently and offer users options for Single Sign-On (SSO). Before 15.0, SaaS customers had to contact the OpenProject support if they wanted custom integrations with their SAML or OpenID connect providers. These new interfaces greatly improve the user experience of adding these providers, which may have been a hurdle at the beginning of your work with OpenProject.
```

### Quick action table headers in project lists for easier navigation 

> With OpenProject 15.0, we are pleased to release another great improvement for our project lists: Clicking on the table headers in a project list now gives you a quick action menu that not only allows you to sort in descending or ascending order, but also to filter or change, remove or add a column. You can still find the same actions in the top-right menu button, but now these actions are now much quicker to access. [Read more about the configuration for project lists in our user guide](../../user-guide/projects/project-lists/#configure-project-lists-view).

Reviewer: ```suggestion
With OpenProject 15.0, we are pleased to release another great improvement for our project lists: Clicking on the table headers in a project list now gives you a quick action menu that not only allows you to sort in descending or ascending order, but also to filter or change, remove or add a column. You can still find the same actions in the top-right menu button, but now these actions are much quicker to access. [Read more about the configuration for project lists in our user guide](../../user-guide/projects/project-lists/#configure-project-lists-view).
```

- Main menu border

All these colors will now be calculated depending on the brightness of the respective background to ensure a high-enough contrast. Font colors will either be black or white, and border colors will be set only if there is a bright background. [Read more about design options for Enterprise customers in our system admin guide](../../system-admin-guide/design/).

### Reduce manual cleanup when adding a custom field to a type – no more auto-applying to all projects

With OpenProject 14.6, we released a feature that allows you to enable or disable a custom field for multiple projects at once. This is why with OpenProject 15.0, we remove the automation to apply a new custom field to all projects where the respective type is activated. This reduces manual cleanup in case you did not want to activate the new custom field in all projects. If you do want that, you can use the feature we introduced last release and go to *Administration > Custom Fields* and click on the 'Add projects' button.

[Read more about custom fields in our system admin guide](../../system-admin-guide/custom-fields/).

### Improved navigation clarity – 'My account' is renamed to 'Account settings'

> In the personal menu that can be accessed by clicking on your avatar, we renamed 'My account' to 'Account settings', in order to give you a clearer understanding that this menu item contains settings. It now also differs more clearly from 'My Page' and 'My Activities‘, which provide personal data instead of settings.

Reviewer: ```suggestion
In the personal menu that can be accessed by clicking on your avatar, we renamed 'My account' to '[Account settings](../../user-guide/account-settings/)', in order to give you a clearer understanding that this menu item contains settings. It now also differs more clearly from 'My Page' and 'My Activities‘, which provide personal data instead of settings.
```
